visualizations/frontend/other/TTable.vue

```diff
@@ -374,14 +376,17 @@ export default {
     };
   },
   computed: {
-      isPdf(){
-          return window.location.href.includes('/pdf?')
-      },
-      allInitializationComplete(){
-        const isPagingInitialized = !this.canPageServer || this.endIndex
-        const isModifiableColumnsInitialized = !this.modifiableColumns || this.$store.state.layers.components[this.tag_unique]?.filters?.column_list !== undefined
-        return isPagingInitialized && isModifiableColumnsInitialized
-      },
+    showTableHeader() {
```

Comment:
maybe `hasTableHeader`? 🤷

Reply:
If your updates in chore/table-gaps-refactoring make this a non-issue I vote for that approach instead.